src/object.js

```diff
@@ -41,6 +41,11 @@ function ReflectionObject(name, options) {
      */
     this.name = name;
 
+    /**
+     * Resolved Features.
+     */
+    this.features = null;
```

Comment:
features should be private, since nobody but the runtime should be inspecting them